src/Surfnet/Stepup/Tests/Configuration/Value/AllowedSecondFactorListTest.php

```diff
@@ -109,7 +109,12 @@ public function an_allowed_second_factor_list_contains_the_given_second_factors(
 
         $allowedSecondFactorList = AllowedSecondFactorList::ofTypes($secondFactorTypes);
 
-        foreach ($allowedSecondFactorList as $index => $actualSecondFactorType) {
+        $builtList = [];
+        foreach ($allowedSecondFactorList as $item){
+            $builtList[] = $item;
+        }
+
+        foreach ($builtList as $index => $actualSecondFactorType) {
```

Comment:
This seems overly complicated. Or do I overlook an 'unused variable' warning or another Stan/PHPMD issue here that you are solving?